Formula/o/opensca-cli.rb

```diff
@@ -0,0 +1,31 @@
+class OpenscaCli < Formula
+  desc "OpenSCA is a supply-chain security tool for security researchers and developers"
+  homepage "https://opensca.xmirror.cn"
+  url "https://github.com/XmirrorSecurity/OpenSCA-cli/archive/refs/tags/v3.0.0.tar.gz"
+  sha256 "65633f9a76e00a218abc9f71a487cc1bb93a8951885915bd05337ebbaec52884"
+  license "Apache-2.0"
+  head "https://github.com/XmirrorSecurity/OpenSCA-cli.git", branch: "master"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "23336fc9722d9e3724bc6fba3a8fbdffc3ee65fc85e5d4a26ef2ce12ff307623"
+    sha256 cellar: :any_skip_relocation, arm64_ventura:  "90bb61d499a0efd35030f37416a1e11c59af29b610d567dfebc617f89198194a"
+    sha256 cellar: :any_skip_relocation, arm64_monterey: "94ded6e709f16f11c8e3a8992a89e2046914ab44f993190e1e96189579fa2d06"
+    sha256 cellar: :any_skip_relocation, sonoma:         "f509822b92ddaa9c3fe08e9799c4cfa57c833c5c6eff3434055a92cb0df87ebb"
+    sha256 cellar: :any_skip_relocation, ventura:        "a9f2b23460ab68bca7223a7bc6ea47313113ba0a9bdff407f14a8b889971a775"
+    sha256 cellar: :any_skip_relocation, monterey:       "262ae6d7d7fa6f329db57062db5b91beac74728f9d9772f12f7df24682ae7848"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "9d57bb267ae51fb3916eb6a1bd16a2933621054e669730c2dc1aa91654f57429"
+  end
```

Comment:
This time it's a little bit different; [CI failed](https://github.com/Homebrew/homebrew-core/actions/runs/7220696722/job/19674158227#step:8:232) because the bottles were already published in #155273. The bottles built in this PR were not published so merge queue CI would presumably fail.

We should use the `bottle` block from the job that successfully published the bottles: https://github.com/Homebrew/homebrew-core/actions/runs/7206892394/job/19632615689#step:12:1431

Reply:
Done